src/videojs-http-streaming.js

```diff
@@ -999,6 +999,18 @@ class VhsHandler extends Component {
       audioMedia: audioPlaylistLoader && audioPlaylistLoader.media()
     });
 
+    this.player_.tech_.on('keystatuschange', (e) => {
+      // typically 'output-restricted', but anything other than usable will
+      // result in a failure to play.
+      if (e.status !== 'usable') {
```

Comment:
I think we should be careful here. Since there are various statuses: https://w3c.github.io/encrypted-media/#dom-mediakeystatus , some of which we handle, i.e., `expired`: https://github.com/videojs/videojs-contrib-eme/blob/5c441dca1896fc73dff2eeb855dd8e60f99c5da4/src/eme.js#L121-L126

It might be better to only blacklist on specific statuses.

Reply:
switched this to only exclude on output-restricted.